### PR TITLE
Disable sdv.databroker.v1 by default

### PR DESCRIPTION
**Pull request by argerus**
- Disable sdv.databroker.v1 by default. It can be enabled by supplying a flag at startup.
- Change databroker-cli to use kuksa.val.v1 by default.

### CODE REVIEW

File: databroker-cli/src/kuksa_cli.rs [hunk 1]
```diff
@@ -36,13 +36,13 @@ const TIMEOUT: Duration = Duration::from_millis(500);
...
     ("get", "<PATH> [[PATH] ...]", "Get signal value(s)"),
-    ("set", "<PATH> <VALUE>", "Set actuator signal"),
+    ("actuate", "<PATH> <VALUE>", "Set actuator signal"),
     (
         "subscribe",
         "<QUERY>",
         "Subscribe to signals with QUERY, if you use kuksa feature comma separated list",
     ),
-    ("feed", "<PATH> <VALUE>", "Publish signal value"),
+    ("publish", "<PATH> <VALUE>", "Publish signal value"),
```
Comment (lukasmittag) [inline databroker-cli/src/kuksa_cli.rs:45]: provide?

File: databroker-cli/src/sdv_cli.rs [hunk 2]
```diff
@@ -42,7 +42,7 @@ const CLI_COMMANDS: &[(&str, &str, &str)] = &[
         "<QUERY>",
         "Subscribe to signals with QUERY, if you use kuksa feature comma separated list",
     ),
-    ("feed", "<PATH> <VALUE>", "Publish signal value"),
+    ("publish", "<PATH> <VALUE>", "Publish signal value"),
```
Comment (lukasmittag) [inline databroker-cli/src/sdv_cli.rs:45]: provide?

File: databroker/src/grpc/server.rs [hunk 3]
```diff
@@ -165,23 +174,35 @@ where
         info!("Authorization is not enabled.");
     }
 
-    builder
-        .add_service(
+    let kuksa_val_v1 = {
+        if apis.contains(&Api::KuksaValV1) {
+            Some(kuksa::val::v1::val_server::ValServer::with_interceptor(
```
Comment (lukasmittag) [inline databroker/src/grpc/server.rs:179]: val_server can maybe cause confusion with old kuksa_val_server in c++?

**Review by lukasmittag — COMMENTED** [review]
Some minor naming discussions

File: databroker/src/grpc/server.rs (hunk 3 above)
Comment (argerus) [reply databroker/src/grpc/server.rs:179]: This was not changed as part of this commit. It is referencing generated code, which having a protobuf service definition like this:

```proto
service VAL {
  ...
}
```

will generate code `<camel case service name>_server::<camel case service name>Server`, i.e. `val_server::ValServer`.

File: databroker-cli/src/kuksa_cli.rs (hunk 1 above)
Comment (argerus) [reply databroker-cli/src/kuksa_cli.rs:45]: Yes, that crossed my mind as well. I'm not against it, but my idea is that `publish` is that act of publishing a single signal value, while "provide" might be better suited for describing the act of continuously providing a signal. "publish" would then be the logical counterpart to "subscribe".

I suppose we will have a discussion around this in the API design rounds. I just couldn't stand having `feed` still there :smiling_face_with_tear:

**Review by lukasmittag — APPROVED** [review]
See the comments above. I agree with them an we can merge this.

**Pull request merged**